syntax = "proto3";

message Traffic_Light {
    int32 color = 1;
    double confidence = 2;
    int32 light_pt_x = 3;
    int32 light_pt_y = 4;
    int32 height = 5;
    int32 width = 6;
}

message TrafficLightlist {
    double timestamp = 1;
    int32 traffic_light_num = 2;
    repeated Traffic_Light trafficlights = 3;
}
